src/client/index.ts

```diff
@@ -1,6 +1,6 @@
-import { Skill } from '../model/cc-proxy/Skill';
```

Comment:
The content of this file is not used anymore...
It should be replaced by: `export './components/learning-spine-browser.js';

If it is not updated now, it should be removed the next commit ;)